lib/routers/user.js

```diff
@@ -26,6 +27,7 @@ router.use((req, res, next) => {
     .then(() => {
       return Invitation.query()
         .where('email', 'iLike', req.user.profile.email)
+        .where('createdAt', '>', moment().utc().startOf('day').subtract(7, 'days'))
```

Comment:
This will expire at the end of the 7th day, not the exact time of day the invite was created.

This matches the way we display expiry on the invitations list, but possibly/probably not how the token expiry works. At least this way when it says "Expiring today" in the establishment admin's invitation list, it will still be visible to the invited user, who can then click the link and potentially get an "invitation expired" message.

The other option is to change the invitation list to show the exact time that the invite expires...

Reply:
I'd remove the `startOf('day')` so that it only ever shows known valid invitations.

It doesn't _really_ matter if that doesn't correspond exactly to the invitations list since no person would ever be able to see both.